### ISSUE TITLE
I0002: TICc2650 does not respond to commands

### ISSUE THREAD

**Issue by dovydas-girdvainis** [Hahn-Schickard/org.openhab.binding.zigbee#1]
TICc2650 (SusLight) has no ZigBeeThingHandler, thus it can not recieve any commands or update it's status to OH2. 
Possible cause - during thing initialization phase in ZigBeeThingHandler class nodes are not deserialized so there information is not loaded, but doing this causes OH2 to show twice the ammount of exisiting channels.

**Comment by dovydas-girdvainis** [comment]
When OH2 is launched and ZigBee binding has finished the starting procedure TICc2650 is shown as ONLINE after refershing the paperUI, but when commands are issued throught paperUI's controll pannel it is quickly changed to OFFLINE. 
Logs show: 
```
[INFO] [me.event.ThingStatusChangedEvent] - 'zigbee:device:722bdaa4:00124b0007e17305' changed from ONLINE to OFFLINE
```

**Comment by dovydas-girdvainis** [comment]
This issue has been solved in 588dc0ef27ea113970cd3cf130aef2ce046e0d71 commit.
